client/assets/components/document/document_mail/document_mail.html

```diff
@@ -1,6 +1,6 @@
 <div class="detail vertical grid-content">
   <!-- show head field linked with a url. -->
-  <h3 class="head-field" ng-if="doc.subject"><a href="#"
+  <h3 class="head-field" ng-if="doc.subject"><a href="http://asfmail.lucidworks.io/"
```

Comment:
@jakemannix @gsingers Can I use the field 'parent_s'? I am not sure what this field means, but it seems right for the ones I sampled..